### ISSUE TITLE
Very verbose llama-server logging by default since recent update

### ISSUE THREAD

**Issue by AInteriorB** [ollama/ollama#16897]
### What is the issue?

Hi, and thanks for the great work!

After updating to the latest version, journald gets flooded with per-request logs from the underlying `llama-server`, even DEBUG is disabled:

```
slot launch_slot_: id 4 | task -1 | sampler chain: logits -> penalties -> ...
slot update_slots: id 4 | task 80 | new prompt, n_ctx_slot = 28160, ...
slot print_timing: id 4 | task 80 | prompt eval time = ...
srv  update_slots: all slots are idle
```

Is it intended that this much gets logged by default? If there's a supported way to reduce it that I missed, I'd appreciate a pointer. Thanks!

### Relevant log output

```shell
Jun 25 10:38:02 ai4 ollama[1636975]: slot get_availabl: id  4 | task -1 | selected slot by LCP similarity, sim_best = 1.000 (> 0.100 thold), f_keep = 0.651
Jun 25 10:38:02 ai4 ollama[1636975]: slot launch_slot_: id  4 | task -1 | sampler chain: logits -> penalties -> ?dry -> ?top-n-sigma -> top-k -> ?typical -> ?top-p -> ?min-p -> ?xtc -> temp-ext -> dist
Jun 25 10:38:02 ai4 ollama[1636975]: slot launch_slot_: id  4 | task -1 | sampler params:
Jun 25 10:38:02 ai4 ollama[1636975]:         repeat_last_n = 64, repeat_penalty = 1.100, frequency_penalty = 0.000, presence_penalty = 0.000
Jun 25 10:38:02 ai4 ollama[1636975]:         dry_multiplier = 0.000, dry_base = 1.750, dry_allowed_length = 2, dry_penalty_last_n = 28160
Jun 25 10:38:02 ai4 ollama[1636975]:         top_k = 40, top_p = 1.000, min_p = 0.000, xtc_probability = 0.000, xtc_threshold = 0.100, typical_p = 1.000, top_n_sigma = -1.000, temp = 0.000
Jun 25 10:38:02 ai4 ollama[1636975]:         mirostat = 0, mirostat_lr = 0.100, mirostat_ent = 5.000, adaptive_target = -1.000, adaptive_decay = 0.900
Jun 25 10:38:02 ai4 ollama[1636975]: slot launch_slot_: id  4 | task 3794 | processing task, is_child = 0
Jun 25 10:38:02 ai4 ollama[1636975]: slot process_sing: id  0 | task -1 | saving idle slot to prompt cache
Jun 25 10:38:02 ai4 ollama[1636975]: slot process_sing: id  1 | task -1 | saving idle slot to prompt cache
Jun 25 10:38:02 ai4 ollama[1636975]: slot process_sing: id  2 | task -1 | saving idle slot to prompt cache
Jun 25 10:38:02 ai4 ollama[1636975]: slot process_sing: id  3 | task -1 | saving idle slot to prompt cache
Jun 25 10:38:02 ai4 ollama[1636975]: slot update_slots: id  4 | task 3794 | new prompt, n_ctx_slot = 28160, n_keep = 4, task.n_tokens = 69
Jun 25 10:38:02 ai4 ollama[1636975]: slot update_slots: id  4 | task 3794 | Checking checkpoint with [0, 64] against 0...
Jun 25 10:38:02 ai4 ollama[1636975]: slot update_slots: id  4 | task 3794 | restored context checkpoint (pos_min = 0, pos_max = 64, n_tokens = 65, n_past = 64, size = 1.524 MiB)
Jun 25 10:38:02 ai4 ollama[1636975]: slot update_slots: id  4 | task 3794 | cached n_tokens = 64, memory_seq_rm [64, end)
Jun 25 10:38:02 ai4 ollama[1636975]: slot update_slots: id  4 | task 3794 | cached n_tokens = 65, memory_seq_rm [65, end)
Jun 25 10:38:02 ai4 ollama[1636975]: slot init_sampler: id  4 | task 3794 | init sampler, took 0.04 ms, tokens: text = 69, total = 69
Jun 25 10:38:02 ai4 ollama[1636975]: slot print_timing: id  4 | task 3794 | prompt eval time =      23.60 ms /     5 tokens (    4.72 ms per token,   211.85 tokens per second)
Jun 25 10:38:02 ai4 ollama[1636975]: slot print_timing: id  4 | task 3794 |        eval time =     217.72 ms /    38 tokens (    5.73 ms per token,   174.53 tokens per second)
Jun 25 10:38:02 ai4 ollama[1636975]: slot print_timing: id  4 | task 3794 |       total time =     241.33 ms /    43 tokens
Jun 25 10:38:02 ai4 ollama[1636975]: slot print_timing: id  4 | task 3794 |    graphs reused =       3750
Jun 25 10:38:02 ai4 ollama[1636975]: slot      release: id  4 | task 3794 | stop processing: n_tokens = 106, truncated = 0
Jun 25 10:38:02 ai4 ollama[1636975]: srv  update_slots: all slots are idle
Jun 25 10:38:02 ai4 ollama[1636975]: [GIN] 2026/06/25 - 10:38:02 | 200 |   802.01326ms |    x.x.x.x | POST     "/v1/chat/completions"
```

### Ollama version

0.30.10

**Comment by anishesg** [comment]
Took a look, https://github.com/ollama/ollama/pull/16899 should address it.

**Comment by rick-github** [comment]
> Is it intended that this much gets logged by default? 

The logging is processed by the server to extract information about how the runner is allocating memory.  Selective filtering would need to be applied in the ollama server to reduce the amount that actually ends up in the ollama server log.

**Comment by anishesg** [comment]
Closed PR #16899 after review feedback - need to verify what log level actually shows scheduler-critical messages before reducing verbosity. The current level 4 is intentionally set to keep offload/memory lines visible for the scheduler.

**Comment by danmackinlay** [comment]
Confirming this on macOS, so it is not journald-specific.

macOS 26.5.2, Apple Silicon, ollama 0.32.8 installed via Homebrew and run from its launchd agent. `OLLAMA_DEBUG` is unset and ollama's own logger is at INFO — the log file held 13 `level=INFO` lines and 1 `level=WARN` — yet the runner is still spawned with `--log-verbosity 4`.

Composition  from a workload that only ever calls `/v1/embeddings`:

| share | source |
|---|---|
| ~95% | `slot ...` and `srv update_slots: all slots are idle`, from `llama-server` |
| ~5% | ollama's own `[GIN]` access log, one line per request |

For embeddings that 95% is pure noise: there is no sampler chain and no generation to inspect, just one pooled forward pass per request.

Steady-state cost here is about 20 MB/day. The file reached 387 MB before I noticed it (No rotation for these brew installs)

Deriving `--log-verbosity` from `OLLAMA_DEBUG`, or exposing it at all, would resolve this.
